client/src/fluid/Fluid.ml

```diff
@@ -5417,13 +5417,17 @@ let pasteOverSelection (data : clipboardContents) (astInfo : ASTInfo.t) :
         ->
           (* Paste into a string, to take care of newlines.
            * Note: pasting before an open quote
-           * places the caret one unit left of the text end *)
```

Comment:
It seems like putting `caret one unit left of the text end` is intended. But not sure what it is for?

Reply:
That seems like it's commenting on the bug you're fixing (as opposed to documenting expected behaviour), so you can just remove the comment.